schema-derivatives-options.graphql

```diff
@@ -1269,36 +1323,27 @@ type ActiveAccount @entity {
 # Note: A position snapshot is not necessary because the position is opened and closed with no other interaction
 # All of the data we would get from a snapshot is already available in the position.
 type Position @entity {
-  " { Account address }-{ Pool address }-{ Position Side }-{ Counter } "
+  " { Account address }-{ Option id }-{ Counter } "
   id: Bytes!
 
   ##### Metadata #####
 
   " The liquidity pool this options contract belongs to "
   pool: LiquidityPool!
 
-  " The type of option this position is. e.g. put/call "
-  type: OptionType!
+  " Reference to the option taken "
+  option: Option!
 
   " Account that owns this position"
   account: Account!
 
-  " The asset the contract was created for (generally the same as the underlying asset in the pool) "
-  asset: Token!
```

Comment:
imo we should keep asset in the position, i know you can grab it from the option, but it is standard across all our schemas to have asset

Reply:
Sure, I'm guessing it should be set to underlying token then?

Reply:
Changed at 1c8014a8203907d0c383d658576f26965836ad9b

Reply:
Changed at 1c8014a8203907d0c383d658576f26965836ad9b